data/i18n/hi/hi.toml

```
@@ -188,7 +188,63 @@ other = "Stack Overflow"
 other = "फोरम"
 [community_events_calendar]
 other = "घटनाक्रम कैलेंडर"
+[community_youtube_name]
+other = "YouTube"
 
 # UI elements
 [ui_search_placeholder]
 other = "खोजें"
+
+[deprecation_title]
+other = "आप इस कुबेरनेटेस संस्करण के लिए प्रलेखन देख रहे हैं:"
+
+[docs_version_current]
+other = "(यह प्रलेखन)"
+
+[docs_version_latest_heading]
+other = "नवीनतम संस्करण"
+
+[docs_version_other_heading]
+other = "पुराने संस्करण"
+
+[end_of_life]
+other = "जीवनकाल का समापन:"
+
+[envvars_heading]
+other = "एनवायरनमेंट वैरिएबल"
+
+[error_404_were_you_looking_for]
+other = "क्या आप यह ढूँढ रहे थें:"
+
+[examples_heading]
+other = "उदाहरण"
+
+[inline_list_separator]
+other = ","
+
+[latest_release]
+other = "नवीनतम रिलीज़:"
+
+[options_heading]
+other = "विकल्प" 
+
+[post_create_issue]
+other = "इशू बनाएं"
+
+[previous_patches]
+other = "पिछले पैच:"
+
+[seealso_heading]
+other = "यह भी देखें"
+
+[subscribe_button]
+other = "सब्स्क्राइब करें"
+
+[synopsis_heading]
+other = "संक्षेप में"
+
+[thirdparty_message]
+other = """यह खंड तीसरे पक्ष की परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन परियोजनाओं के लिए जिम्मेदार नहीं हैं। यह पृष्ठ <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF website guidelines</a> का अनुसरण करते हुए परियोजनाओं को वर्णानुक्रम में सूचीबद्ध करता है। इस सूची में कोई नयी परियोजना जोड़ने से पहले यह <a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a> पृष्ट पढ़के ही परिवर्तन करें।"""
```

nit: I'd localize “content guide“.

---

Thanks @sftim , I will do the needful

---

तीसरे पक्ष की परियोजनाओं  is a literal transliteration. We're looking at third party in the context of these projects/products not being developed/maintained by the K8s community. The below suggestion fits better, IMO. WDYT?

```suggestion
other = """यह खंड अन्य पक्ष परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन परियोजनाओं के लिए जिम्मेदार नहीं हैं। यह पृष्ठ <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF website guidelines</a> का अनुसरण करते हुए परियोजनाओं को वर्णानुक्रम में सूचीबद्ध करता है। इस सूची में कोई नयी परियोजना जोड़ने से पहले यह <a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a> पृष्ट पढ़के ही परिवर्तन करें।"""
```

---

Yes I think using `अन्य` will be much better and it gives a clear idea of the context. I will the do the needful and push a new commit based on the feedback. Thank you.